Markdown_to_TeX/markdown_compile.py

```diff
@@ -200,25 +187,28 @@ def _latex_string(self, sections):
         #If no sections are detected then errors out.
         if len(sections) == 0:
             print("***** Error: No sections detected in your markdown file.")
+            logging.error("No sections detected in your markdown file.")
             sys.exit(1)
         
         #If no abstract it errors out.  Shame on you...
         if "ABSTRACT" not in sections:
-            print("***** Error: No abstract in markdown.  Make a section titled #ABSTRACT")
-            sys.exit(1)
+            print("----- Warning: No abstract in markdown.  Most documents should have an abstract.")
+            logging.warning("No abstract in markdown.  Most documents should have an abstract.")
         if self.verbose:
+            logging.debug(str(sections))
             print(sections)
 
         #Storage for latex formatted strings for abstract and sections.
         abstract_str = ""
         sections_str = ""
 
         #Get abstract data
-        for txt in sections["ABSTRACT"]["text"].split("\n"):
-            if txt.strip() == "":
-                continue
-            abstract_str += txt
-            abstract_str += (r"\\") + ("\n")
+        if "ABSTRACT" in sections:
+            for txt in sections["ABSTRACT"]["text"].split("\n"):
+                if txt.strip() == "":
+                    continue
+                abstract_str += txt
```

Comment:
I've had issues in the past attempting to use math operators to manipulate strings. A drop-in replacement for `+=` would be
```suggestion
                abstract_str.append(txt)
```

Reply:
Its just shorthand for string concatenation.  I personally am not a fan of the append syntax outside of the context of arrays.  The efficiency difference is also largely negligible for strings of this length.